test/credentials_test.dart

```diff
@@ -226,4 +226,119 @@ Future<void> main([List<String>? args]) async {
+
+    baasTest('Manual test 2 (resetPassword). Take recieved token from the received email and confirm the user', (configuration) async {
+      // Enter valid token and tokenId from the received email
+      String token = "3e23e2e689fe1fdbbb51d3c090a216a4195a4f0a004a578787618d9dd39c791f4169511ee3820e4b6fa2bfdc14430f1e58356223d78e3bed3c86042c3a91a4db";
+      String tokenId = "6278cecd9106aa5b645999ba";
+
+      final app = App(configuration);
+      final authProvider = EmailPasswordAuthProvider(app);
+      await authProvider.confirmUser(token, tokenId);
+      final user = await loginWithRetry(app, Credentials.emailPassword(validUsername, strongPassword));
+      expect(user, isNotNull);
+    }, appName: "emailConfirm", skip: "It is a manual test");
+
+    baasTest('Manual test 3 (resetPassword). Reset user password email', (configuration) async {
+      final app = App(configuration);
+      final authProvider = EmailPasswordAuthProvider(app);
+      await authProvider.resetPassword(validUsername);
+    }, appName: "emailConfirm", skip: "It is a manual test");
+
+    baasTest('Manual test 4 (resetPassword). Take recieved token from the email and complete resetting new password', (configuration) async {
+      // Make sure you have recieved an emails with hyperlink ResetPassword.
+      // Find the token and tokenId in the query parameters of the link received in the email and enter them in the following variables.
+      String token = "fb485146b15497209a9d1b67128ae29199cdff2c26f389e0ee5d52ae6bc4228e5738b29256eade976e24767804bfb4dc68198075e3a461cd4f73864901fb09be";
+      String tokenId = "6272619334f5b3a770a7dc2c";
+
+      final app = App(configuration);
+      final authProvider = EmailPasswordAuthProvider(app);
+      String newPassword = "RWE@#EDE";
+      await authProvider.completeResetPassword(newPassword, token, tokenId);
+      final user = await loginWithRetry(app, Credentials.emailPassword(validUsername, strongPassword));
+      expect(user, isNotNull);
+    }, appName: "emailConfirm", skip: "Run this test manually after test 1 and after setting token and tokenId");
+  });
+
+  // The tests in this group are for manual testing, since they require interaction with mail box.
+  // Please enter a valid data in the variables under comments.
+  // The email should be a valid and existing one in order you to be able to receive automatic emails.
+  // Run first two steps (test 1 and test 2) to create and confirm the user.
+  // Before running test 2 set the variables with token details received as link query parameters in the confirmation email.
+  // Then run test 3 to call reset password function configured in the cloud.
+  // Test 3 will check also whether the new password is applied and the old password is rejected.
+  // Run the last test 4 to call resetFunc without passing additional arguments.
+  group("Manual test: Email/Password - call reset password function with new password", () {
```

Comment:
Why do this need to be manual? As far as I can tell we only have a manual step for confirming the user which can be achieved either by using automatic user confirmation or by using function confirmation.

Reply:
You are right. I will change it.

Reply:
Done